.github/workflows/test.yml

```diff
@@ -111,10 +178,27 @@ jobs:
       - name: Make workerd binary executable
         run: chmod +x /tmp/workerd
 
-      - name: Build Wrangler + Miniflare & dependencies
-        run: pnpm build --filter miniflare --filter wrangler
+      - name: Download generated types
+        uses: actions/download-artifact@v4
+        with:
+          name: generated-snapshot
+          path: /tmp/test-types
+
+      - name: Create and install workers-types package
+        run: |
+          cp /tmp/test-types/oldest/* /tmp/test-types
+          cat > /tmp/test-types/package.json << EOF
+          {
+              "name": "@cloudflare/workers-types",
+              "version": "4.20250405.0"
+            }
+          EOF
+          pnpm add /tmp/test-types -w
+
+      - name: Build Wrangler, Miniflare, Vitest, Vite & dependencies
+        run: pnpm build --filter miniflare --filter wrangler --filter @cloudflare/vitest-pool-workers --filter @cloudflare/vite-plugin
```

Comment:
This step should not be necessary because Turbo should build the dependencies as part of the test:ci step below.